phpunit.xml.dist

```diff
@@ -10,7 +10,8 @@
          stopOnFailure="false"
          stopOnIncomplete="false"
          stopOnSkipped="false"
-         verbose="false">
+         verbose="false"
+         printerClass="NyanCat\PHPUnit\ResultPrinter">
```

Comment:
This will make the output on Travis unnecessarily verbose (and not nyan-pretty). +2 for the rest of the PR.

Reply:
I changed it so the CI is not nyan